api/src/accounts.rs

```diff
@@ -519,17 +516,20 @@ impl Account {
     /// Find a resource associated with an account
     fn find_resource(
         &self,
-        resource_type: &StructTag,
+        tag: &StructTag,
```

Comment:
Nit: `resource_type` or `value_type` would be more meaningful than `tag`

Reply:
Done